config/components.php

```diff
@@ -140,9 +140,16 @@
 	'search' => function (
 		App $kirby,
 		Collection $collection,
-		string $query = '',
-		$params = []
+		string $query = null,
+		array|string $params = []
 	): Collection|bool {
+		$query = trim($query ?? '');
+
+		// empty search query
+		if (empty($query) === true) {
```

Comment:
Could we also cover the "shorter than minlength" check from line 168 here? Or rather delete this additional check and move the `clone $collection` in line 164 after the check in line 168? This avoids duplicate code.